kivy/core/image/img_pil.py

```diff
@@ -102,7 +102,11 @@ def load(self, filename):
 
     @staticmethod
     def save(filename, width, height, fmt, pixels, flipped=False):
-        image = PILImage.fromstring(fmt.upper(), (width, height), pixels)
+        # Support PIL 3+
+        try:
+            image = PILImage.fromstring(fmt.upper(), (width, height), pixels)
+        except:
```

Comment:
ok, on further review, from string has simply been removed from Pillow. Just replace fromstring in this case with `frombytes`. That should work with all mordern versions of Pillow.